src/app/shared/palette/payment/case-payment-history-viewer-field.scss

```diff
@@ -0,0 +1,8 @@
+@import "~govuk_frontend_toolkit/stylesheets/_colours.scss";
+
+.manual-link {
```

Comment:
This should be changed to provided correct styling.

Reply:
This was a copy from CUT library. Can we review this is needed?

Reply:
I cannot see that class used in Payments web components, I'm removing it.